README.md

```diff
@@ -1 +1,185 @@
-# math-magician
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+ <!-- LOGO -->
+
+  <img src="./src/My Logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+<!-- MAIN HEADING -->
+
+  <h3><b>Encarta Math Magician</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Encarta Math Magician <a name="about-project"></a>
+
+> This is a Math Magician app designed to handle and perform mathematical calculations.
+
+## 🛠 Built With <a name="built-with"></a>
+1. React
+2. CSS
+3. JavaScript
+4. Webpack
+### Tech Stack <a name="tech-stack"></a>
+
+> Tech Stack to be updated soon
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JavaScript</a></li>
+    <li><a href="https://reactjs.org/">Webpack</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Perform simple and complex mathematical calculations
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+LIVE DEMO
+
+> Coming soon...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy of the project, use this link:
+> 
+`
+https://github.com/anyars-encarta/math-magician.git
+`
+
+
+<!-- SETUP -->
+### Setup
+
+To setup this project, run this command:
+
+``npm start
+``
+### Prerequisites
+
+1. A Browser (Preferably Google Chrome)
+2. A Code Editor
+3. Internet Connection
+4. Git
+
+<!-- INSTALL -->
+### Install
+
+Install this project with Iroko.
+
+### Usage
+
+To run the project, execute the following command:
+
+``npm start
+``
+### Run tests
+> npm run test
+### Deployment
+
+You can deploy this project using:
+>1.  GitHub Pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
```

Comment:
- [ ] Kindly remove this section description as requested by the previous reviewer.

Reply:
Sorry for the oversight sir